modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -131,7 +131,7 @@ var backendServiceInvoker = function () {
         httpMethodObject.addRequestHeader(header);
         if (IS_OAUTH_ENABLED) {
             var accessToken = getAccessToken();
-            if (!(!accessToken.trim())) {
+            if (accessToken) {
```

Comment:
Noted.
since accessToken is not a boolean use the second option recommended here and few other places like this and updated the PR